### ISSUE TITLE
Create an FAQ to help school districts think twice about registering a .gov domain

### ISSUE THREAD

**Issue by h-m-f-t** [cisagov/get.gov#284]
### Issue description

School districts are technically eligible for .gov, but it's an awkward fit. We should draft an FAQ about this, post it online, and share it with relevant parties.

### Acceptance criteria

- [ ] Create content in our [FAQ doc](https://docs.google.com/document/d/1bfBndWQzSJLzx2WOLKiaLRVmSsl6mxbYzBxBU7R1848/edit#heading=h.sunk4m9if1y)
- [ ] Consider what text should be updated to point to this FAQ

### Additional context

There are states (Utah, California) with laws requiring .gov's use, and it's being interpreted to include schools. There are also misunderstandings about the [SLCGP](https://www.cisa.gov/state-and-local-cybersecurity-grant-program), .gov, and school districts that this could help address.

**Comment by katypies** [comment]
@h-m-f-t - this still seems valuable. Thoughts on trying to address this? I see it also impacting Analyst workflow and communications if there's stronger public guidance.

**Comment by h-m-f-t** [comment]
Still a good idea and we should draft something. The hard part is syncing it with others' equities. Let's keep it open.

**Comment by gabo0oo** [comment]
@h-m-f-t This seems like something product would lead, writing the first draft of what we want, and then a designer could refine it. What do you think? We're also wondering if it's still relevant considering its created date.

**Comment by h-m-f-t** [comment]
@gabo0oo yes, I think Product would lead this. I'd like to leave this issue open, though I think we're likely to update policies to explicitly put school districts out of eligibility before we make this FAQ.
